introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi. My name is Lars Reith. I am Technical Director at SapientRazorfish.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?